source/Halibut/Queue/Redis/RedisPendingRequest.cs

```diff
@@ -145,8 +145,7 @@ public async Task WaitUntilComplete(Func<Task> checkIfPendingRequestWasCollected
                         log.Write(EventType.MessageExchange, "Request {0} was cancelled before a response was received", request);
                         SetResponseNoLock(ResponseMessage.FromException(
                                 request,
-                                new TimeoutException("A request was sent to a polling endpoint, the polling endpoint collected it but the request was cancelled before the polling endpoint responded."),
-                                ConnectionState.Connecting),
+                                new TimeoutException("A request was sent to a polling endpoint, the polling endpoint collected it but the request was cancelled before the polling endpoint responded.")),
                             requestWasCollected: false);
```

Comment:
Just wondering, we have established that the request has been collected (and are now waiting for a response). So why is the variable `requestWasCollected` false?